doc/netplan-tutorial.md

```diff
@@ -225,518 +244,214 @@ network:
-    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
-2: enp5s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
-    link/ether 00:16:3e:13:ae:10 brd ff:ff:ff:ff:ff:ff
-3: netplan-isp: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
-    link/ether 00:16:3e:0c:97:8a brd ff:ff:ff:ff:ff:ff
-```
-
-# Creating a link aggregation
-
-Let's suppose now that you need to configure your system to connect to your
-ISP links via a link aggregation. On Linux you can do that with a `bond`
-virtual interface.
-
-On Netplan, an interface of type `bond` can be created inside a `bonds` mapping.
-
-Now that the traffic will flow through the link aggregation, you will move
-all the addressing configuration to the bond itself.
-
-You can define a list of interfaces that will be attached to the bond. In our
-simple scenario, we have a single one.
-
-Edit the file `/etc/netplan/second-interface.yaml` and make the following changes:
-
-```yaml
-network:
-  version: 2
-  ethernets:
-    netplan-isp-interface:
-      dhcp4: false
-      dhcp6: false
-      match:
-        macaddress: 00:16:3e:0c:97:8a
-      set-name: netplan-isp
-  bonds:
-    isp-bond0:
-      interfaces:
-        - netplan-isp-interface
-      dhcp4: false
-      dhcp6: false
-      accept-ra: false
-      link-local: []
-      addresses:
-        - 172.16.0.1/24
-      routes:
-        - to: default
-          via: 172.16.0.254
-      nameservers:
-        search:
-          - netplanlab.local
-        addresses:
-          - 172.16.0.254
-          - 172.16.0.253
-```
-
-Note that you can reference the interface used in the bond by the name you
-defined for it in the `ethernets` section.
-
-Now use `netplan apply` to apply your changes
-
-```
-netplan apply
-```
-
-Now your system has a new interface called `isp-bond0`. Use the command
-`ip address show isp-bond0` or `netplan status` to check its state:
-
-```
-netplan status isp-bond0
-```
-
-You should see an output similar to the one below:
-
-```
-     Online state: online
-    DNS Addresses: 127.0.0.53 (stub)
-       DNS Search: lxd
-                   netplanlab.local
-
-●  4: isp-bond0 bond UP (networkd: isp-bond0)
-      MAC Address: b2:6b:19:b1:9a:86
-        Addresses: 172.16.0.1/24
-    DNS Addresses: 172.16.0.254
-                   172.16.0.253
-       DNS Search: netplanlab.local
-           Routes: default via 172.16.0.254 (static)
-                   172.16.0.0/24 from 172.16.0.1 (link)
-
-3 inactive interfaces hidden. Use "--all" to show all.
-```
+In this tutorial you learned how to set up a learning environment for Netplan using LXD virtual machines, explored Netplan configuration, including its underlying configuration files, and tried the `netplan set`, `netplan get`, `netplan apply`, and `netplan status` commands. You also used some of the Ethernet configuration options to enable a network interface with DHCP.
```

Comment:
suggestion (non-blocking): We should consider showcasing the `netplan try` command somewhere in the tutorial,in addition to `netplan apply`,  as that seems to be a well received feature by our community.

Reply:
Ack. I'll file an issue for this.